atomic_reactor/plugins/pre_bump_release.py

```diff
@@ -186,25 +186,39 @@ def run(self):
         parser = df_parser(self.workflow.builder.df_path, workflow=self.workflow)
         dockerfile_labels = parser.labels
         labels = Labels(dockerfile_labels)
+        missing_labels = []
 
         component_label = labels.get_name(Labels.LABEL_TYPE_COMPONENT)
-
         try:
             component = dockerfile_labels[component_label]
         except KeyError:
-            raise RuntimeError("missing label: {}".format(component_label))
+            self.log.error("missing label: %s", component_label)
+            missing_labels.append(component_label)
 
         version_label = labels.get_name(Labels.LABEL_TYPE_VERSION)
         try:
             version = dockerfile_labels[version_label]
+            if not version:
+                self.log.error('label is empty: %s', version_label)
+                missing_labels.append(version_label)
         except KeyError:
-            raise RuntimeError('missing label: {}'.format(version_label))
+            self.log.error('missing label: %s', version_label)
+            missing_labels.append(version_label)
 
+        release_label = labels.get_name(Labels.LABEL_TYPE_RELEASE)
         try:
             _, release = labels.get_name_and_value(Labels.LABEL_TYPE_RELEASE)
         except KeyError:
             release = None
 
+        if release is not None and not release:
```

Comment:
this could go in the try/except `else` clause instead. This way you could get `release_label` from the `get_name_and_value` call (no need for the `get_name` call above).

Reply:
yeah will do that